src/Tools/ExternalAccess/FSharp/Classification/FSharpClassificationService.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Classification
+{
+    [ExportLanguageServiceFactory(typeof(IClassificationService), LanguageNames.FSharp), Shared]
```

Comment:
📝 In its current form, this type leaves us with no way to transition F# to an API that doesn't use `IEditorClassificationService`, which means we will not be able to remove that interface.